internal/k8sutils/utils.go

```
@@ -104,6 +104,23 @@ func IsNodeReady(node *corev1.Node) bool {
 	return false
 }
 
+func TpuTopologyToChipCount(topo string) (int, error) {
+	// TODO: Do we need to validate expectedDims? GKE won't run the jobset if this is invalid?
+	split := strings.Split(topo, "x")
+	if split == nil {
+		return -1, fmt.Errorf("invalid topology: %v", topo)
```

Nit: I would return the same error val pair in each error case (i.e. `0`)